userspace/libsinsp/threadinfo.cpp

```diff
@@ -203,57 +199,6 @@ void sinsp_threadinfo::fix_sockets_coming_from_proc() {
 	});
 }
 
-#define STR_AS_NUM_JAVA 0x6176616a
-#define STR_AS_NUM_RUBY 0x79627572
-#define STR_AS_NUM_PERL 0x6c726570
-#define STR_AS_NUM_NODE 0x65646f6e
-
-#define MAX_PROG_HASH_LEN 1024
-
-void sinsp_threadinfo::compute_program_hash() {
-	auto curr_hash = std::hash<std::string>()(m_exe);
-	hash_combine(curr_hash, m_container_id);
```

Comment:
is `hash_combine()` in utils.h used anywhere else?

Reply:
Only here :rofl: